spec/ar2dto/relation_to_dto/attributes_spec.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+RSpec.describe ".to_dto" do
+  describe "attributes" do
```

Comment:
Same comment.

Not sure about the naming of this file and describe block.

I started by creating all the other files which seemed more specific, and the tests inside here were the "remaining" ones.